### PR TITLE
Update sa.cv

### PR DESCRIPTION
**Pull request by frekeeh**
Update sa.csv

I added the last 6 websites that Saudi blocked in May 2017 
below is a link to an article on the websites Saudi blocked after their conflict with Qatar 
https://www.alarabiya.net/ar/saudi-today/2017/05/24/%D9%87%D8%B0%D9%87-%D9%82%D8%A7%D8%A6%D9%85%D8%A9-%D8%A7%D9%84%D9%85%D9%88%D8%A7%D9%82%D8%B9-%D8%A7%D9%84%D9%82%D8%B7%D8%B1%D9%8A%D8%A9-%D8%A7%D9%84%D9%85%D8%AD%D8%AC%D9%88%D8%A8%D8%A9-%D9%81%D9%8A-%D8%A7%D9%84%D8%B3%D8%B9%D9%88%D8%AF%D9%8A%D8%A9.html

### CODE REVIEW

File: lists/sa.csv [hunk 1]
```diff
@@ -398,3 +398,9 @@ http://www.yementodaytv.net/,NEWS,News Media,2016-04-19,citizenlab,
 http://www.youtube.com/verify_controversy?next_url=/watch%3fv%3dqmodvun16q4,NEWS,News Media,2014-04-15,citizenlab,
 http://www.zawadj.net,DATE,Online Dating,2014-04-15,citizenlab,
 http://yawmyat.blogspot.com,NEWS,News Media,2014-04-15,citizenlab,
+https://www.aljazeera.com/,NEWS,News Media,2018-06-14,frekeeh,
```
Comment (hellais) [inline lists/sa.csv]: I would suggest moving this to the global list. We currently have aljazeera.net, but that doesn't seem to be the canonical domain anymore.

@agrabeli thoughts?

Comment (agrabeli) [reply lists/sa.csv]: @hellais We should probably be testing both aljazeera.com and aljazeera.net via the global test list (sorry for seeing this question so late!).

**Comment by agrabeli** [comment]
@hellais @sneft Do you perhaps have a script for automatically adding all the missing slashes (see details on Travis failed checks)?

**Comment by agrabeli** [comment]
I added aljazeera.com to the global test list: #510

**Comment by agrabeli** [comment]
Thank you @frekeeh for the contributions! It seems that this PR has the following new contributions:

https://alaraby.tv/,NEWS,News Media,2018-06-14,frekeeh,
https://www.alaraby.co.uk/portal,NEWS,News Media,2018-06-14,frekeeh,
https://alaraby.tv/,NEWS,News Media,2018-06-14,frekeeh,
http://www.qna.org.qa/,NEWS,News Media,2018-06-14,frekeeh,
http://www.al-watan.com/,NEWS,News Media,2018-06-14,frekeeh,
http://www.raya.com/portal,NEWS,News Media,2018-06-14,frekeeh,
https://www.alarab.qa/,NEWS,News Media,2018-06-14,frekeeh,
https://www.al-sharq.com/,NEWS,News Media,2018-06-14,frekeeh,
http://alaraby.tv/,NEWS,News Media,2018-06-14,frekeeh,
https://www.alaraby.co.uk/portal,NEWS,News Media,2018-06-14,frekeeh,

However, I manually checked each of these URLs and saw that they have already been merged into the SA test list (either as part of older contributions, or as part of more recent contributions by Netalitica researchers). 

I am therefore closing this pull request.

**Pull request closed without merging**